src/app/components/mini.tsx

```diff
@@ -19,7 +19,9 @@ const Mini: React.FC = () => {
                 </span>
                 :{" "}
                 <span
-                  style={{ color: radio.currentlyRx ? "green" : "inherit" }}
+                  style={{
+                    color: radio.currentlyRx ? "lightgreen" : "inherit",
```

Comment:
Unrelated change 😬 I found "green" was too dark against the dark background. Let me know if you'd like this split out to a separate PR instead.

Reply:
Approved